pkg/datapath/linux/config/config.go

```diff
@@ -50,6 +54,8 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "datapath-config")
```

Comment:
Is it possible to move the logging part of this into options validation in the daemon?

Inviting the logging into the config package here increases the likelihood that we end up re-logging the same message again and again any time we make datapath changes. In the node case like this, it'll only be if someone reconfigures the daemon I think; but if we end up adding log messages to the endpointconfig / netdev config then it will print every time someone starts an endpoint, and potentially multiple times as regenerations are requested.

Reply:
> Is it possible to move the logging part of this into options validation in the daemon?

Moved to pkg/datapath/linux/requirements.go, as [suggested by André](https://github.com/cilium/cilium/pull/10264#discussion_r401425558).

> Inviting the logging into the config package here increases the likelihood that we end up re-logging the same message again and again any time we make datapath changes. In the node case like this, it'll only be if someone reconfigures the daemon I think; but if we end up adding log messages to the endpointconfig / netdev config then it will print every time someone starts an endpoint, and potentially multiple times as regenerations are requested.

I understand, I thought adding the log package was maybe not desirable (hence the separate commit).

Reply:
> Moved to pkg/datapath/linux/requirements.go

On second thought, moving to `initEnv()`.